sql/tsql/TSqlLexer.g4

```diff
@@ -107,6 +107,7 @@ GOTO:                                  'GOTO';
 GRANT:                                 'GRANT';
 GROUP:                                 'GROUP';
 HAVING:                                'HAVING';
+HIDDEN_:                               'HIDDEN';
```

Comment:
Probably newly added keywords should be treated as identifiers as well.

Reply:
Sorry, this one I don't understand.

Reply:
I mean this word could be identifiers in some contexts, it's not a reserved keyword. It should be added to `keyword` list: https://github.com/antlr/grammars-v4/blob/master/sql/tsql/TSqlParser.g4#L4016

Reply:
Ok. I've pushed an update.